tests/unit/render/DefaultRender.spec.js

```diff
@@ -17,10 +19,73 @@ describe('Test Class: DefaultRender()', () => {
   });
 
   describe('Test methods', () => {
-    it('Test render() method', () => {
-      const defaultRender = new DefaultRender();
+    describe('Test method: render', () => {
```

Comment:
render can only return an empty array ?

Reply:
yes, because this code is made by plugin :)